### ISSUE TITLE
Project Dead?

### ISSUE THREAD

**Issue by bjay1404** [KaiserSoft/PIA-Tunnel#15]
Is there going to be any more development on this project? I use this software in production, but it requires an again debian image. What features were you working on next? Would you be interested in some sort of shell install file that installs the software on a vanilla Debian 9 or Ubuntu  image?

**Comment by EricTheRed1** [comment]
Yes. It is dead.  The author said he no longer uses it so it is not being developed any longer.

**Comment by bjay1404** [comment]
Has anybody gotten this project working on a fresh install of Ubuntu or Debian? Ubuntu 1604 or Debian 9?
